### PR TITLE
Added doc for eventing-sink test function

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Added doc for eventing-sink test function

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/16840

### CODE REVIEW

File: tests/fast-integration/eventing-test/assets/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Eventing Sink
+
+## Overview
+
+This eventing sink function is used to test the end-to-end flow for Eventing. It is responsible for three tasks:
```
Comment (NHingerl) [inline tests/fast-integration/eventing-test/assets/README.md:5]:
```suggestion
Use the `eventing-sink` Function to test the end-to-end flow for Eventing. 
It performs three tasks:
```

If this is about a Serverless Function, we should [capitalize](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#camel-case) it.

File: tests/fast-integration/eventing-test/assets/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. Forward requests to publish events to event-publisher-proxy.
```
Comment (NHingerl) [inline tests/fast-integration/eventing-test/assets/README.md:6]: If the order of list elements is important, keep the numbered list. 
If the order does not matter, use a bullet list.

File: tests/fast-integration/eventing-test/assets/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+2. Save all received events in memory.
+3. Return saved events to users.
+
+## Usage
+
+### Forward requests to publish events to event-publisher-proxy.
+
+To publish an event to event-publisher-proxy, send a Http request with params `send=true` and body:
```
Comment (NHingerl) [inline tests/fast-integration/eventing-test/assets/README.md:14]:
```suggestion
To publish an event to `event-publisher-proxy`, send an HTTP request with parameter `send=true` and the following body:
```

File: tests/fast-integration/eventing-test/assets/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    data: {
+        headers: {}, // Event headers
+        payload: {}  // Event payload
+    },
+}
+```
+
+### Fetch an event received by eventing-sink from a backend.
+
+To fetch an event received by eventing-sink, send a Http request with params `eventid=<id>`. The response contains:
```
Comment (NHingerl) [inline tests/fast-integration/eventing-test/assets/README.md:28]:
```suggestion
To fetch an event received by `eventing-sink`, send an HTTP request with parameter `eventid=<id>`. The response contains:
```

File: tests/fast-integration/eventing-test/assets/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+{
+    url: `http://eventing-event-publisher-proxy.kyma-system/publish`,
+    data: {
+        headers: {}, // Event headers
+        payload: {}  // Event payload
+    },
+}
+```
+
+### Fetch an event received by eventing-sink from a backend.
```
Comment (NHingerl) [inline tests/fast-integration/eventing-test/assets/README.md:26]:
```suggestion
### Fetch an event received by `eventing-sink` from a backend
```

Headlines don't need a full stop

File: tests/fast-integration/eventing-test/assets/README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+## Overview
+
+This eventing sink function is used to test the end-to-end flow for Eventing. It is responsible for three tasks:
+1. Forward requests to publish events to event-publisher-proxy.
+2. Save all received events in memory.
+3. Return saved events to users.
+
+## Usage
+
+### Forward requests to publish events to event-publisher-proxy.
```
Comment (NHingerl) [inline tests/fast-integration/eventing-test/assets/README.md:12]:
```suggestion
### Forward requests to publish events to `event-publisher-proxy`
```

Headlines don't need a full stop.

**Review by NHingerl — COMMENTED** [review]
came across this one and guessed it's ready for review :D

File: tests/fast-integration/eventing-test/assets/README.md (hunk 2 above)
Comment (mfaizanse) [reply tests/fast-integration/eventing-test/assets/README.md]: Updated it to use bullet list.

**Review by NHingerl — APPROVED** [review]

**Pull request merged**